static/js/util/profile_edit.js

```diff
@@ -48,8 +48,9 @@ export function boundRadioGroupField(keySet: string[], label: string, options: O
     if (obj.helper) {
       helper = `${obj.helper}`;
     }
+    let labelId = `gender-option-${obj.label}`;
```

Comment:
This doesn't work properly if `obj.label` has spaces in it, such as "Other/Prefer not to say".